pkg/infoschema/infoschema_test.go

```diff
@@ -953,13 +953,13 @@ func TestEnableInfoSchemaV2(t *testing.T) {
 	// Test the @@tidb_enable_infoschema_v2 variable.
 	tk.MustQuery("select @@tidb_schema_cache_size").Check(testkit.Rows("0"))
 	tk.MustQuery("select @@global.tidb_schema_cache_size").Check(testkit.Rows("0"))
-	require.Equal(t, variable.SchemaCacheSize.Load(), int64(0))
+	require.Equal(t, variable.SchemaCacheSize.Load(), uint64(0))
 
 	// Modify it.
-	tk.MustExec("set @@global.tidb_schema_cache_size = 1024")
-	tk.MustQuery("select @@global.tidb_schema_cache_size").Check(testkit.Rows("1024"))
-	tk.MustQuery("select @@tidb_schema_cache_size").Check(testkit.Rows("1024"))
-	require.Equal(t, variable.SchemaCacheSize.Load(), int64(1024))
+	tk.MustExec("set @@global.tidb_schema_cache_size = 1073741824")
```

Comment:
Please check if there's test cases like "10MB"

Reply:
it the pkg/sessionctx/variable/sysvar_test.go we add 10KB test